src/utils/sort.ts

```diff
@@ -128,12 +148,12 @@ export function sortObjectList<T extends {}>(
   const result = items.map((_, index) => index);
 
   if (sortBy) {
-    sorter(result, items, sortBy, order === 'descending');
+    sorter(result, items, sortBy, groupBy, order === 'descending');
+  } else if (groupBy) {
+    sorter(result, items, groupBy, undefined, order === 'descending');
   }
 
-  if (groupBy && groupBy !== sortBy) {
-    sorter(result, items, groupBy);
-  }
+  console.log('RESULT: ', result);
```

Comment:
I guess we don't need this log?

Reply:
Damn... there is always one hiding somewhere...